CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ Hello contributors! Welcome to raylib!
 Do you enjoy raylib and want to contribute? Nice! You can help with the following points:
 
 - `C programming` - Can you write/review/test/improve the code? 
-- `Documentation/Tutorials/Example` - Can you write some tutorial/example?
+- `Documentation/Tutorials/Example` - Can you write some tutorials/examples?
```

Comment:
> Alternatively we could do
> "Can you write a tutorial/example?"


I like that version better

Secondly, the `Documentation/Tutorials/Example` can probably be changed to `Examples` (plural) as well

(imo)

Reply:
I actually like "Can you write a tutorial/example?" better as well.